src/samples.rs

```diff
@@ -0,0 +1,91 @@
+use std::collections::BTreeMap;
+
+use reddsa::frost::{
+    redjubjub::{
+        keys::{
+            KeyPackage, PublicKeyPackage, SecretShare, SigningShare,
+            VerifiableSecretSharingCommitment, VerifyingShare,
+        },
+        round1::{NonceCommitment, SigningCommitments},
+        round2::SignatureShare,
+        Identifier, SigningPackage, VerifyingKey,
+    },
+    redpallas::{
+        frost::{Element, Scalar},
+        Ciphersuite, Field, Group,
+    },
+};
+
+use reddsa::frost::redjubjub::JubjubBlake2b512;
+
+type C = JubjubBlake2b512;
+
+fn element1() -> Element<C> {
```

Comment:
I believe `Element<C>` could more simply be `JubjubGroup::Element`. Same for all the other uses of Scalar, Ciphersuite, Field, Group. I don't see a reason to redefine the types already defined in `reddsa`